pkg/storage/leveldbstore/recovery.go

```diff
@@ -35,13 +36,18 @@ func (p *pendingTx) Unmarshal(bytes []byte) error {
 // Recover attempts to recover from a previous
 // crash by reverting all uncommitted transactions.
 func (s *TxStore) Recover() error {
+	logger := log.NewLogger("node").WithName("tx_store").Register() // "node" - copies the node.LoggerName in order to avoid circular import.
```

Comment:
maybe the name should have "recovery" in it?

Reply:
Done.